tests/integration/instance-initializers/axe-component-test.js

```diff
@@ -146,120 +139,114 @@ test('audit should log any violations found', function(assert) {
   });
 
   const logSpy = sandbox.spy(Logger, 'error');
-  const component = Component.create({});
-
-  // In order for the audit to run, we have to act like we're not in testing
-  Ember.testing = false;
-
-  component.audit();
+  this.render(hbs`{{#axe-component}}{{content}}{{/axe-component}}`);
 
   assert.ok(logSpy.calledOnce);
-
-  // Turn testing mode back on to ensure validity of other tests
-  Ember.testing = true;
 });
 
 
 test('audit should do nothing if no violations found', function(assert) {
+  initialize();
+  initializeViolationsHelper();
+
   stubA11yCheck(sandbox, { violations: [] });
 
   const logSpy = sandbox.spy(Logger, 'error');
-  const component = Component.create({});
-
-  component.audit();
+  this.render(hbs`{{#axe-component}}{{content}}{{/axe-component}}`);
 
   assert.ok(logSpy.notCalled);
 });
 
 /* Component.axeCallback */
 
 test('axeCallback receives the results of the audit', function(assert) {
-  const results = { violations: [] };
-  const axeCallbackSpy = sandbox.spy();
-  const component = Component.create({
-    axeCallback: axeCallbackSpy
-  });
+  initialize();
```

Comment:
Seems like this is always used, can we move it to the `beforeEach` instead of in each test?